### PR TITLE
repo-manifests.rst:Text changes for clarity and consistency.

### PR DESCRIPTION
**Pull request by jeremiah**
Signed-off-by: Jeremiah C. Foster <jeremiah@jeremiahfoster.com>

- The word 'repo' in the text can be confusing, therefor I thought I'd spell it out when we mean 'repository' and leave the word 'repo' to represent the tool repo.
- Some of the text gets a little turgid at times so I tried to simplify it as much as possible.

### CODE REVIEW

**Review by tobsan — APPROVED** [review]

**Comment by tobsan** [comment]
@jeremiah Jenkins fails your commit because the underline of your heading is too short. Please update :)

File: docs/articles/baseplatform/repo-manifests.rst
Comment (GordanM) [inline docs/articles/baseplatform/repo-manifests.rst]: Ah yes, here is the reason of the failure.

File: docs/articles/baseplatform/repo-manifests.rst
Comment (GordanM) [inline docs/articles/baseplatform/repo-manifests.rst]: ... and here :).

**Review by GordanM — APPROVED** [review]
Thanks a lot @jeremiah looks much better now!

**Review by breakreturn — APPROVED** [review]
Looks good. Given that the breaking "typos" are fixed I'm all for merging.

**Comment by jeremiah** [comment]
"All checks have passed" w00t

**Comment by breakreturn** [comment]
@jeremiah OK, thanks for fixing that! There's a conflict that you should resolve before we can merge though. Pull the latest master and rebase your text-changes branch to that and resolve whatever the conflict is. Also, to keep the history clean and to work according our recommendations, please squash your two commits (the original one and the fixes) into one and force push that to your branch.

The last step is technically not required but considered best practice by the maintainers, it's a slight hassle but we all do it frequently while in the review process :)

**Comment by jeremiah** [comment]
Will do!

**Comment by jeremiah** [comment]
Squished, squashed, smooshed.

**Comment by breakreturn** [comment]
@jeremiah Nice. Thanks for sticking with our guidelines!

**Comment by jeremiah** [comment]
Sure thing! I'm thick but it penetrates the skull eventually!

**Comment by breakreturn** [comment]
My motto = embrace thickness

When all of us do that, we'll have the best software on the planet :)

Thanks for the contribution.

**Pull request merged**
